src/entrypoints/main/router/validator/index.tsx

```diff
@@ -10,6 +10,7 @@ const ValidatorRouter: FC<RouteComponentProps> = ({ match: { url } }) => (
   <Suspense fallback={null}>
     <Switch>
       <Route exact path={url} component={pages.validator} />
+      <Route path={`${url}/:resource`} component={pages.validator} />
```

Comment:
Dette er i bunn og grunn samme route. Det er ikke nødvendig å lage en ny. Jeg foreslår at vi enten bruker query params eller optional path param for resource.
```
<Route path={`${url}/:resource?`} component={pages.validator} />
```